packages/esm-patient-medications-app/src/order-basket/order-basket-search/drug-search.component.tsx

```diff
@@ -21,8 +21,8 @@ export default function OrderBasketSearch({ onSearchResultClicked }: OrderBasket
         size="lg"
         light={isTablet}
         value={searchTerm}
-        placeholder={t('searchFieldPlaceholder', 'Search for a drug or orderset (e.g. "Aspirin")')}
-        labelText={t('searchFieldPlaceholder', 'Search for a drug or orderset (e.g. "Aspirin")')}
+        placeholder={t('drugSearchFieldPlaceholder', 'Search for a drug or orderset (e.g. "Aspirin")')}
+        labelText={t('drugSearchField', 'Search Field')}
```

Comment:
We can keep both placeholder and labelText the same, since the labelText is not visible, no need to create a new key in the translation file and this is also very generic too.
Thanks!